.travis.yml

```diff
@@ -1,3 +1,4 @@
 language: node_js
 node_js:
   - "node"
+  - "0.10"
```

Comment:
This version is used to verify that synctos builds properly on the full range of Node.js versions used by the organization.